compiler/rustc_feature/src/active.rs

```diff
@@ -620,6 +620,9 @@ declare_features! (
     /// Allows arbitrary expressions in key-value attributes at parse time.
     (active, extended_key_value_attributes, "1.50.0", Some(78835), None),
 
+    /// Allows users to provide classes for fenced code block using `class:classname`.
+    (active, custom_code_classes_in_docs, "1.50.0", Some(79483), None),
```

Comment:
I don't think this will land in 1.50, so:

```suggestion
    (active, custom_code_classes_in_docs, "1.51.0", Some(79483), None),
```